src/main/java/cceclipseplugin/editor/DocumentManager.java

```diff
@@ -204,18 +208,18 @@ public void handleNotification(Notification n) {
 	 * 
 	 * @param fileId
 	 *            fileId to patch; this is mainly used for passing to clientCore
-	 * @param filePath
+	 * @param absolutePath
 	 *            absolute file path; used as key in editorMap, and patches.
 	 * @param patches
 	 *            the list of patches to apply, in order.
 	 */
-	public void applyPatch(long fileId, String filePath, List<Patch> patches) {
+	public void applyPatch(long fileId, String absolutePath, String workspaceRelativePath, List<Patch> patches) {
```

Comment:
Make sure include a javadoc for the workspaceRelativePath parameter that specifies that you need a path that includes the filename